### ISSUE TITLE
buildBoostrap is not agnostic of the platform (win64 vs ubuntu1404)

### ISSUE THREAD

**Issue by SulivanDotEu** [sensiolabs/SensioDistributionBundle#296]
Hello,

I'm using my host machine (win32) to manage my code source and dependencies with composer. I work on my project through PhpStorm, manage the source files with GIT thanks to Cmder and update dependencies with composer. Everytime the source code changes, PhpStorm will upload it on my VM by SFTP. It was working fine until composer 1.2.2 maybe 1.2.1. Since then, when I install/update dependancies with composer, the script ```php vendor/sensio/distribution-bundle/Sensio/Bundle/DistributionBundle/Resources/bin/build_bootstrap.php``` won't work if runned from my host machine. But it will work from the VM machine. Here is the result when I run it from windows : 

```
Warning: require(/var/www/dev/app\..\vendor\symfony\symfony\src\Symfony\Component\ClassLoader\ApcClassLoader.php): failed to open stream: No such file or directory in /var/www/dev/app/bootstrap.php.cache on line 2883

Fatal error: require(): Failed opening required '/var/www/dev/app\..\vendor\symfony\symfony\src\Symfony\Component\ClassLoader\ApcClassLoader.php' (include_path='.:/usr/share/php:/usr/share/pear:') in /var/www/dev/app/bootstrap.php.cache on line 2883
```

**It works fine if I run the command directly in my VM.**
Here is my version of sensio/distribution-bundle:

```
           { "name": "sensio/distribution-bundle",
            "version": "v2.3.22",
            "target-dir": "Sensio/Bundle/DistributionBundle",
            "source": {
                "type": "git",
                "url": "https://github.com/sensiolabs/SensioDistributionBundle.git",
                "reference": "98bdda791e7c2dfb5fd55781e69a4b00e4f751a6"
            },
            "dist": {
                "type": "zip",
                "url": "https://api.github.com/repos/sensiolabs/SensioDistributionBundle/zipball/98bdda791e7c2dfb5fd55781e69a4b00e4f751a6",
                "reference": "98bdda791e7c2dfb5fd55781e69a4b00e4f751a6",
                "shasum": ""
            },
            "require": {
                "symfony/framework-bundle": "~2.2"
            },
            "type": "symfony-bundle",
            "extra": {
                "branch-alias": {
                    "dev-master": "2.3.x-dev"
                }
            },
            "autoload": {
                "psr-0": {
                    "Sensio\\Bundle\\DistributionBundle": ""
                }
            },
            "notification-url": "https://packagist.org/downloads/",
            "license": [
                "MIT"
            ],
            "authors": [
                {
                    "name": "Fabien Potencier",
                    "email": "fabien@symfony.com"
                }
            ],
            "description": "The base bundle for the Symfony Distributions",
            "keywords": [
                "configuration",
                "distribution"
            ],
            "time": "2015-06-05 22:32:08"
        },
```

I'm not 100% sure it is related to the bundle or composer itself. To me, all windows host might be impacted by this. It's quite new to me. It's happening on all my symfony project since a few days or weeks.

Hope it will help.

**Comment by HeahDude** [comment]
Hello @SulivanDotEu, you're problem seems related to the APC extension not available in your VM, you can either remove the loader from your `app.php` or try to change its configuration.

Anyway I'm closing here since it does not look related to this repo.

Feel free to reopen if I'm wrong. Thanks!

**Comment by SulivanDotEu** [comment]
Hello @HeahDude, unfortunately, this is not related to APC. As I seed, I'm speaking about the file generated by the script : ´´´php vendor/sensio/distribution-bundle/Sensio/Bundle/DistributionBundle/Resources/bin/build_bootstrap.php´´´ which is related I think to this repo.

If I run the script from windows, the file **app/bootstrap.php.cache** is **not** correct.
If I run the script from linux, the file **app/bootstrap.php.cache** is correct.

The file "/var/www/dev/app\..\vendor\symfony\symfony\src\Symfony\Component\ClassLoader\ApcClassLoader.php" do exist on my VM, the problem is the **DIRECTORY_SEPARATOR**.

This issue is quite new so that why I created a ticket.

**Comment by fabpot** [comment]
@SulivanDotEu The good news is that it has already been fixed in symfony/symfony#20676

**Comment by SulivanDotEu** [comment]
@fabpot awesome, thanks for the reply.
